src/Port.js

```diff
@@ -0,0 +1,14 @@
+class Port {
+  constructor(name) {
+    this.name = name;
+    this.ships = [];
+  }
+  addShip(ship) {
+    this.ships.push(ship);
+  }
+  removeShip(ship) {
```

Comment:
Its a common convention to leave a clear line between each method to make the file easier to read at a glance:
```
  }

  addShip(ship) {
    this.ships.push(ship);
  }

  removeShip(ship) {
```